differ/differ_test.go

```diff
@@ -1451,3 +1453,116 @@ func TestProcessClustersWeeklyDigest(t *testing.T) {
 	zerolog.SetGlobalLevel(zerolog.WarnLevel)
 	kafkaNotifier = originalNotifier
 }
+
+// ---------------------------------------------------------------------------------------
```

Comment:
why this comment?

Reply:
There are separators like this at other places in code - I thought it would be reasonable to place it here, since the added test is for Service Log functionality (which has nothing to do with rest of the `differ.go`)